enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj

```diff
@@ -46,9 +50,9 @@
                                 (when-not (= syncable-attributes (:login_attributes user))
                                           {:login_attributes syncable-attributes})
                                 (when (not= first-name old-first-name)
-                                          {:first_name first-name})
+                                      {:first_name first-name})
```

Comment:
This indentation looks busted

Reply:
oh you're right. the indentation of the whole function was already messed up actually, I'll put up a quick PR to fix that

Reply:
https://github.com/metabase/metabase/pull/27467